File: LitQuestAPI/LitQuestAPI/Models/Review.cs
using System;
using System.Collections.Generic;

namespace LitQuestAPI.Models;

public partial class Review
{
    public int Reviewid { get; set; }

    public int Userid { get; set; }

    public string Text { get; set; }

    public float Rating { get; set; }

    public string Bookid { get; set; } = null!;

    public virtual User User { get; set; } = null!;
}

File: view/src/models/Review.js
import React, { Component } from "react";

class Review extends Component {
  constructor(props) {
    super(props);
    this.state = {
      reviewid: 0,
      userid: 0,
      text: "",
      rating: 0.0,
      bookid: "",
      user: null,
    };
  }

  render() {
    return (
      <div>
        <p>Review ID: {this.state.reviewid}</p>
        <p>User ID: {this.state.userid}</p>
        <p>Text: {this.state.text}</p>
        <p>Rating: {this.state.rating}</p>
        <p>Book ID: {this.state.bookid}</p>
        <p>User: {this.state.user}</p>
      </div>
    );
  }
}

export default Review;